src/pages/patch/PatchRestartModal.tsx

```diff
@@ -122,14 +147,15 @@ export const PatchRestartModal: React.FC<PatchModalProps> = ({
     >
       {!loading && patchBuildVariants && (
         <>
-          <TreeSelect
-            onChange={setValidStatus}
-            state={validStatus}
-            tData={statusesTreeData}
-            inputLabel="Tasks Selected: "
-            dataCy="patch-status-filter"
-            width="50%"
-          />
+          <Row>
+            <TaskStatusFilters
```

Comment:
I think now that there are 2 filters we should make the dropdowns be a little larger.  its hard to tell whats in them. 
![image](https://user-images.githubusercontent.com/4605522/99842574-15e18500-2b3e-11eb-8621-80a0bec6ac26.png)